### DISCUSSION TITLE
Time Series Forecasting - From Moez - Apr 18 2022

### DISCUSSION THREAD

**Discussion by msdumanlc** [pycaret/pycaret#2876] [General]
Following the steps provided in posting: https://moez-62905.medium.com/time-series-in-python-10502f9fac2a
Using Python 3.8.10, Tried pycaret[full] and --pre pycaret with same results.

Q1) On Model training and Selection the documentation states the Least Angle Regression (LAR) is the best model.  However when I run it under Juypter-Labs I get Orthogonal Matching Pursuit (OMP)?  Why?  

Q2) When plotting the final graph the future years are always 0?  Looks like future label values are off?

I have attached an .IPYNB file.  Also a screen shot of final results with the issue.
Let me know what I am doing wrong?  Thanks in advance.

[Time Series Forecasting with PyCaret Regression Module.zip](https://github.com/pycaret/pycaret/files/9376957/Time.Series.Forecasting.with.PyCaret.Regression.Module.zip)
.

**Comment by celestinoxp** [comment]
@moezali1

**Reply by msdumanlc** [reply]
I have researched this further.  Seems to be an issues with predictions_future = predict_model(final_best, data=future_df)
Either the predict_model is not scaling correctly on return or a step was not presented on the website https://moez-62905.medium.com/time-series-in-python-10502f9fac2a .  On the website the Label values are large.  However if you look at my version (see zip file in prior post) the Label values are small.  Why?  This is why the I can not duplicate the plot correctly.  
Using latest JupyterLab and 
    python: 3.8.10 (tags/v3.8.10:3d8993a, May  3 2021, 11:48:03) [MSC v.1928 64 bit (AMD64)]
executable: c:\users\mduman\.pyenv\pyenv-win\versions\3.8.10\python.exe
   machine: Windows-10-10.0.19044-SP0

PyCaret required dependencies:
                 pip: 22.2.2
          setuptools: 56.0.0
             pycaret: 3.0.0.rc3
             IPython: 8.4.0
          ipywidgets: 8.0.0rc2
                tqdm: 4.64.0
               numpy: 1.21.6
              pandas: 1.4.3
              jinja2: 3.1.2
               scipy: 1.8.1
              joblib: 1.1.0
             sklearn: 1.1.2
                pyod: Installed but version unavailable
            imblearn: 0.9.1
   category_encoders: 2.5.0
            lightgbm: 3.3.2
               numba: 0.55.2
            requests: 2.28.1
          matplotlib: 3.5.3
          scikitplot: 0.3.7
         yellowbrick: 1.4
              plotly: 5.10.0
             kaleido: 0.2.1
         statsmodels: 0.13.2
              sktime: 0.11.4
               tbats: Installed but version unavailable
            pmdarima: 1.8.5
              psutil: 5.9.1

![image](https://user-images.githubusercontent.com/109091381/191079516-794d7d03-f528-43fc-b4e8-89f318818513.png)
![image](https://user-images.githubusercontent.com/109091381/191080034-9f16a914-ac26-4a55-a5c6-236602a3a499.png)

Appreciate anything you can suggest.

@moezali1
